src/main/java/com/google/sps/servlets/ReasonServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that sends reason for use to /reason. (will change to write to database)*/
+@WebServlet("/reason")
```

Comment:
Similar comment as above, at the end, we should only have two pages. 

In this task https://github.com/googleinterns/step67-2020/issues/9, when user clicks submit button, this will direct user to the second page, and pass the reason to the second page. Since there is already a "/main-page" page in this project, consider using it and directing to this page with the reason coded in the parameter.

Reply:
Here as well I will focus on the user journey and also I will separate these commits.